docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb

```diff
@@ -0,0 +1,1196 @@
+    "train_ds = train_ds.map(lambda x: {\"text\": x[\"title\"] + \" \" + x[\"abstract\"]}, remove_columns=[\"title\", \"abstract\"])\n",
+    "valid_ds = valid_ds.map(lambda x: {\"text\": x[\"title\"] + \" \" + x[\"abstract\"]}, remove_columns=[\"title\", \"abstract\"])\n",
+    "test_ds = test_ds.map(lambda x: {\"text\": x[\"title\"] + \" \" + x[\"abstract\"]}, remove_columns=[\"title\", \"abstract\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "G2wkHugwyXDn"
+   },
+   "source": [
+    "## Configuring and training the pipeline\n",
+    "\n",
+    "As we have seen in [previous tutorials](https://www.recogn.ai/biome-text/documentation/tutorials/1-Training_a_text_classifier.html#explore-the-training-data), a *biome.text* [`Pipeline`](https://www.recogn.ai/biome-text/documentation/basics.html#pipeline) consists of tokenizing the input, extracting text features, applying a language encoding (optionally) and executing a task-specific head in the end. In *biome.text* the pre-trained transformers by Hugging Face are treated as a text feature, just like the *word* and *char* feature.\n",
+    "\n",
+    "In this section we will configure and train 3 different pipelines to showcase the usage of transformers in *biome.text*."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ETEfq79HEltM"
+   },
+   "source": [
+    "### Fine-tuning the transformer\n",
+    "\n",
+    "In our first pipeline we follow the common approach to use pretrained transformers in classification tasks. It consists of fine-tuning the transformer weights and using a special token as pooler in the end. In our configuration the former step means setting the `trainable` parameter in the *transformers* features to `True`. The downside of fine-tuning is that most of the pre-trained transformers are relatively big and require dedicated hardware to be fine-tuned. For example, in this tutorial we will use `distilroberta-base`, a [distilled version](https://github.com/huggingface/transformers/tree/master/examples/distillation) of RoBERTa with a total of ~80M parameters.\n",
+    "\n",
+    "We also need to specify the maximum number of input tokens `max_length` supported by the pretrained transformer. If you are sure that your input data does not exceed this limit, you can skip this parameter.\n",
+    "\n",
+    "With BERT like models, like RoBERTa, a special [CLS] token is added as first token to each input. It is pretrained to effectively represent the entire input and can be used as pooler in the head component. Many BERT like models pass this token through a non-linear tanh activation layer that is part of the pretraining. If you want to use these pretrained weights you have to use the `bert_pooler` together with the corresponding `pretrained_model`. We will fine-tune these weights as well (setting `require_grad` to `True`) and add a little dropout.\n",
```

Comment:
Super small kind of typo: I would write "with BERT-like models such as RoBERTa", to avoid the two 'like' so close. The new context lines are cool!

Reply:
:+1: done